src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoMend.kt

```diff
@@ -21,14 +23,23 @@ internal object AutoMend : Module(
     private val autoThrow by setting("AutoThrow", true)
     private val autoSwitch by setting("AutoSwitch", true)
     private val autoDisable by setting("AutoDisable", false, { autoSwitch })
+    private val cancelNearby by setting("CancelNearby", NearbyMode.OFF)
+
+    private val pauseNearbyRadius by setting("NearbyRadius", 10, 1..100, 1, { cancelNearby != NearbyMode.OFF })
```

Comment:
max range of 100 is too great and unnecessary. max range of 8 should be enough. Since this is for avoiding mending someone else's armor in your hole.